docs/modules/ROOT/pages/development.adoc

```diff
@@ -267,33 +267,46 @@ end
 === Auto-correct
 
 The auto-correct can help humans automatically fix offenses that have been detected.
-It's necessary to define an `autocorrect` method that returns a lambda
-https://github.com/whitequark/parser/blob/master/lib/parser/rewriter.rb[rewriter]
-with the corrector where you can give instructions about what to do with the
+It's necessary to `extend Autocorrector`.
+The method `add_offense` yields a corrector object that is a thin wrapper on
+https://www.rubydoc.info/gems/parser/Parser/Source/TreeRewriter[parser's TreeRewriter]
+to which you can give instructions about what to do with the
 offensive node.
 
 Let's start with a simple spec to cover it:
 
 [source,ruby]
 ----
-it 'autocorrect `!a.empty?` to `a.any?` ' do
+  it 'corrects `!a.empty?`' do
```

Comment:
I see this code block and the ones that follow are indented by some margin (probably because you copied them straight from some source file.

Reply:
Fixed